packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -99,6 +99,7 @@ const DEFAULT_OPTIONS = {
 
   // Timestamp
   timestampColor: DEFAULT_SUBTLE,
+  timestampConfig: 'relative', // 'absolute'
```

Comment:
Can it be `timestampFormat` instead? Config is a bit too bit and vague.

```suggestion
  timestampFormat: 'relative', // 'absolute'
```

Reply:
lol